surfsense_backend/app/routes/documents_routes.py

```
@@ -191,36 +190,74 @@ async def process_file_in_background(
-            await add_received_file_document(
-                session,
-                filename,
-                docs,
-                search_space_id
-            )
+                # Clean up the temp file
+                import os
+                try:
+                    os.unlink(file_path)
+                except:
+                    pass
```

Ruff (0.11.9)

Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

Replace with `contextlib.suppress(Exception)`

(SIM105)